### PR TITLE
Add Fork Digest Helper

### PR DESCRIPTION
**Pull request by nisdas**
- [x] Adds in compute fork digest helper.

### CODE REVIEW

File: beacon-chain/core/helpers/signing_root.go [hunk 1]
```diff
@@ -99,3 +100,21 @@ func computeForkDataRoot(version []byte, root []byte) ([32]byte, error) {
...
+//
+// Spec pseudocode definition:
+//	def compute_fork_digest(current_version: Version, genesis_validators_root: Root) -> ForkDigest:
+//    """
+//    Return the 4-byte fork digest for the ``current_version`` and ``genesis_validators_root``.
+//    This is a digest primarily used for domain separation on the p2p layer.
+//    4-bytes suffices for practical separation of forks/chains.
+//    """
+//    return ForkDigest(compute_fork_data_root(current_version, genesis_validators_root)[:4])
+func ComputeForkDigest(version []byte, root []byte) ([4]byte, error) {
```
Comment (terencechain) [inline beacon-chain/core/helpers/signing_root.go]: As a preventive measure, perhaps add a simple test so next person has to update the test if `ComputeForkDigest` implementation changes again.

Comment (rauljordan) [inline beacon-chain/core/helpers/signing_root.go]: Let's rename the arg to genesisValidatorsRoot, more informative that way

Comment (nisdas) [reply beacon-chain/core/helpers/signing_root.go]: done

Comment (nisdas) [reply beacon-chain/core/helpers/signing_root.go]: done

**Pull request merged**